### PR TITLE
Add Hotaru armor, another Japanese-Inspired Armor

### PR DESCRIPTION
**Pull request by novaepee**
## About The Pull Request

image

MAGENTA
image

image
## Why It's Good For The Game

More armor! Permission from ilikesaying but Griffer-249 to port this armor to TGMC.
## Changelog
:cl: novaepee, OrcaOrca the spriter
add: Add Hotaru armor, another Japanese-Inspired Armor! find it in the style line tab!
/:cl:

### CODE REVIEW

**Review by Kuro020 — APPROVED** [review]
I'm not pink, I'm 100% MAGENTA

**Comment by ilikesaying** [comment]
It's Riotrooping time
image

File: code/game/objects/machinery/vending/marine_vending.dm [hunk 1]
```diff
@@ -1027,7 +1027,7 @@
 			/obj/item/clothing/head/modular/marine/eva = -1,
 			/obj/item/clothing/head/modular/marine/eva/skull = -1,
 			/obj/item/clothing/head/modular/marine/helljumper = -1,
-			/obj/item/clothing/head/modular/marine/ranger = -1,
```
Comment (Lumipharon) [inline code/game/objects/machinery/vending/marine_vending.dm:1030]: wuh?

**Review by TiviPlus — CHANGES_REQUESTED** [review]
Gelida changes?
Baneling?
I think you may have broken some stuff

**Comment by novaepee** [comment]
my git got fucked

**Pull request closed without merging**